meson.build

```diff
@@ -201,6 +201,9 @@ if get_option('plugin_uefi')
     conf.set('HAVE_FWUP_GET_BGRT_INFO', '1')
   endif
   efivar = dependency('efivar')
+  if fwup.version().version_compare('< 11')
+    conf.set('LIBFWUP_LIBRARY_VERSION', fwup.version())
```

Comment:
can't we just use `conf.set_quoted()` and avoid the printf?

Reply:
Not if we want to keep the same format as is used in fwupdate upstream currently: https://github.com/rhboot/fwupdate/blob/master/linux/include/fwup-version.h.in